xml/hardening.xml

```diff
@@ -3445,6 +3445,327 @@ Account Expires: Never</screen>
+     access method is only meant for emergency access to the system and should
+     never be used for everyday system administration tasks.
+    </para>
+
+    <note>
+     <para>
+      The steps shown here are tailored towards PC architectures (&x86; and
+      &x86-64;). On architectures like &power; different terminal
+      device names than <literal>tty1</literal> may be used. Be careful not to
+      lock yourself out completely by specifying wrong terminal device names.
+      You can determine the device name of the terminal you are currently
+      logged into by running the <literal>tty</literal> command. Be careful
+      not to do this in a virtual terminal like via SSH or in a graphical
+      session (device names <filename>/dev/pts/<replaceable>N</replaceable></filename>)
+      but only from an actual login terminal reachable via
+      <keycombo><keycap function="alt"/><keycap>F<replaceable>N</replaceable></keycap></keycombo>.
+     </para>
+    </note>
+
+    <procedure>
+     <title>Restricting root logins on local TTYs</title>
+     <step>
+      <para>
+       Ensure that the PAM stack configuration file <filename>/etc/pam.d/login</filename>
+       contains the <literal>pam_securetty</literal> module in the <literal>auth</literal> block:
+      </para>
+      <screen>auth     requisite      pam_nologin.so
+ auth     [user_unknown=ignore success=ok ignore=ignore auth_err=die default=bad] pam_securetty.so noconsole
+ auth     include        common-auth</screen>
+      <para>
+        This will include the <literal>pam_securetty</literal> PAM module during the
+        authentication process on local consoles to allow root only to login on
+        tty devices that are listed in the file <filename>/etc/securetty</filename>.
+      </para>
+     </step>
+     <step>
+      <para>
+       Remove all entries from <filename>/etc/securetty</filename> except one.
+       This limits the access to TTY devices for root.
+      </para>
+     <screen>#
+# This file contains the device names of tty lines (one per line,
+# without leading /dev/) on which root is allowed to login.
+#
+tty1</screen>
+     </step>
+     <step>
+      <para>
+       Check whether logins to other terminals will be rejected for
+       &rootuser;. A login on <literal>tty2</literal>, for example, should be
+       rejected right away without even the root password being queried.
+       Also make sure that you can still successfully login to
+       <literal>tty1</literal> and thus that &rootuser; is not locked out of
+       the system completely.
+      </para>
+     </step>
+    </procedure>
+
+    <important>
+     <para>
+      Avoid adding the <literal>pam_securetty</literal> PAM module to the
+      <filename>/etc/pam.d/common-auth</filename> file. This change would
+      break <command>su</command> and <command>sudo</command>, however,
+      because these tools would then also reject &rootuser; authentications.
```

Comment:
```suggestion
      Do not add the <literal>pam_securetty</literal> module to the
      <filename>/etc/pam.d/common-auth</filename> file. This would
      break the <command>su</command> and <command>sudo</command> commands,
      because these tools would then also reject &rootuser; authentications.
```

Reply:
This might be better as a [warning] -- do you agree, @tomschr?

Reply:
Yes, we can do that in the fine-tuning part. :wink: